### DISCUSSION TITLE
Adding a comment on code in a PR now always starts a review

### DISCUSSION THREAD

**Discussion by lerouxb** [community/community#7581] [Pull Requests]
For the last few days every time I leave a comment on code in a PR's "Files changed" tab it leaves the comment AND starts a review. Then the next comment I add becomes a review comment.

I don't want to create a review because I don't always want to give the impression that my comments are things I expect changed - they could just be comments or nitpicks and I don't want to give the impression that it is a list of demands.

Previously I could just keep clicking "Add single comment" without a review starting. This seems to be a recent regression.

**Comment by jefflward** [comment]
Possible duplicate https://github.com/github/feedback/discussions/7481
